### PR TITLE
Chore/stylelint config sharable

### PR DESCRIPTION
**Pull request by literat**
## Description

Preparation for the Stylelint upgrade.

### CODE REVIEW

File: configs/stylelint-config-spirit/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+To install `stylelint-config-spirit` in your project, you will need to run the following command using [Yarn][yarn]:
+
+```bash
+yarn add stylelint-config-spirit
+```
+
+## 🚀 Usage
+
+```js
+// stylelint.config.js
```
Comment (crishpeen) [inline configs/stylelint-config-spirit/README.md:18]: maybe
```suggestion
// .stylelintrc.mjs
```
like we actually use it?

File: configs/stylelint-config-spirit/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# stylelint-config-spirit
+
+> StyleLint configuration and preset for Spirit Design System
```
Comment (crishpeen) [inline configs/stylelint-config-spirit/README.md:3]:
```suggestion
> Stylelint configuration and preset for Spirit Design System
```

File: configs/stylelint-config-spirit/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "stylelint-config-spirit",
+  "description": "StyleLint configuration for Spirit Design System",
```
Comment (crishpeen) [inline configs/stylelint-config-spirit/package.json:3]:
```suggestion
  "description": "Stylelint configuration for Spirit Design System",
```

File: configs/stylelint-config-spirit/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "0.0.0",
+  "license": "MIT",
+  "keywords": [
```
Comment (crishpeen) [inline configs/stylelint-config-spirit/package.json:6]: add `stylelint`?

**Review by crishpeen — APPROVED** [review]

File: configs/stylelint-config-spirit/README.md (hunk 1 above)
Comment (literat) [reply configs/stylelint-config-spirit/README.md:18]: This file can have multiple names, I will reword it to something like `Stylelint Configuration File`

File: configs/stylelint-config-spirit/package.json (hunk 4 above)
Comment (literat) [reply configs/stylelint-config-spirit/package.json:6]: Added.

File: configs/stylelint-config-spirit/package.json (hunk 3 above)
Comment (literat) [reply configs/stylelint-config-spirit/package.json]: Fixed.

File: configs/stylelint-config-spirit/README.md (hunk 2 above)
Comment (literat) [reply configs/stylelint-config-spirit/README.md]: Fixed.
